Gems/LmbrCentral/Code/Source/Shape/EditorBoxShapeComponent.cpp

```diff
@@ -70,16 +70,14 @@ namespace LmbrCentral
         EditorBaseShapeComponent::Activate();
         m_boxShape.Activate(GetEntityId());
         AzFramework::EntityDebugDisplayEventBus::Handler::BusConnect(GetEntityId());
-        AzToolsFramework::BoxManipulatorRequestBus::Handler::BusConnect(
-            AZ::EntityComponentIdPair(GetEntityId(), GetId()));
-        AzToolsFramework::ShapeManipulatorRequestBus::Handler::BusConnect(
-            AZ::EntityComponentIdPair(GetEntityId(), GetId()));
+        const AZ::EntityComponentIdPair entityComponentIdPair(GetEntityId(), GetId());
+        AzToolsFramework::BoxManipulatorRequestBus::Handler::BusConnect(entityComponentIdPair);
+        AzToolsFramework::ShapeManipulatorRequestBus::Handler::BusConnect(entityComponentIdPair);
```

Comment:
Don't we need to disconnect from these buses in `EditorBoxShapeComponent::Deactivate` ?